### PR TITLE
My articles

### PR DESCRIPTION
**Pull request by EstebanIndiveri**
#### Summary
Added a new My Articles Screen
used the same component by Home and My Favorites.
Added to redux store without api now

#### Android
Captura de Pantalla 2021-08-04 a la(s) 18 10 38

#### Ios

Captura de Pantalla 2021-08-04 a la(s) 17 49 35
Captura de Pantalla 2021-08-04 a la(s) 17 51 42
Captura de Pantalla 2021-08-04 a la(s) 17 56 00

### CODE REVIEW

File: src/app/components/AppNavigator/navigator.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import SignUp from '@authScreens/SignUp';
 import Home from '@screens/Home';
 import Profile from '@screens/Profile';
 import NewArticle from '@screens/NewArticle';
+import MyArticles from '@app/screens/MyArticles';
```
Comment (santiagorobra) [inline src/app/components/AppNavigator/navigator.tsx:14]: @screens/MyArticles

File: src/app/screens/MyArticles/index.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import i18next from 'i18next';
+import ScreenWithLoader from '@components/ScreenWithLoader';
+import { MyArticlesState, State } from '@interfaces/reduxInterfaces';
+import { Article } from '@interfaces/articlesInterface';
+import { ListKeyExtractor } from '@interfaces/miscelanious';
+import MyActiclesActions from '@redux/myArticles/actions';
+import CustomText from '@components/CustomText';
+
+import './i18n';
+import ArticleItem from '../Home/components/ArticleItem';
```
Comment (john07289) [inline src/app/screens/MyArticles/index.tsx:13]: @screens/Home/components/ArticleItem';

File: src/app/components/AppNavigator/navigator.tsx (hunk 1 above)
Comment (EstebanIndiveri) [reply src/app/components/AppNavigator/navigator.tsx]: changed

File: src/app/screens/MyArticles/index.tsx (hunk 2 above)
Comment (EstebanIndiveri) [reply src/app/screens/MyArticles/index.tsx]: changed

**Review by andyOlarte514 — APPROVED** [review]

**Pull request merged**